scene/2d/skeleton_2d.cpp

```diff
@@ -268,18 +661,120 @@ void Skeleton2D::_notification(int p_what) {
 			_update_transform();
 		}
 
+		set_process(true);
+		set_physics_process(true);
```

Comment:
In this way the `Skeleton` is always processing? What if I don't want to use IK? I guess that we need a way to enable / disable (automatically / manually), somehow.

Reply:
I would need to check the code and re-familiarize myself with what is going on to know for sure. We might be able to make it where the processing is set when a SkeletonModificationStack2D is set, which would make it where those who do not use IK do not pay for it.